ios-base/DroidKaigi 2020/Extentions/UIImageView+Ex.swift

```diff
@@ -1,20 +1,73 @@
 import UIKit
 
 extension UIImageView {
-    func loadImage(url: URL?) {
+    class Cache {
+        private var cache = [String: UIImage]()
```

Comment:
I think it's better to  add cache limit or  use NSCache.

Reply:
Good catch! I completely forgot about `NSCache`...